crates/accountsdb-rabbitmq/Cargo.toml

```diff
@@ -30,6 +30,8 @@ solana-accountsdb-plugin-interface = "1.9.4"
 solana-logger = "1.9.4"
 solana-program = "1.9.4"
 solana-sdk = "1.9.4"
+spl-token = "3.2.0"
+reqwest = "*"
```

Comment:
We should make this an explicit version, depending on * can break the build without warning.